### ISSUE TITLE
Do not delete session history when reloading page 

### ISSUE THREAD

**Issue by khalo-sa** [scalableminds/chatroom#131]
Hi,

is it possible to keep the chat history of a session visible after a page reload?
In my case, the chat window becomes completely blank if I had a conversation with the bot, and then reload the page.

Thank you!

**Comment by hotzenklotz** [comment]
No, unfortunately session recovery is not yet implemented in the Chatroom. It can be done though, using the Rasa Tracker API . See https://rasa.com/docs/rasa/api/http-api/#operation/getConversationTracker

**Comment by digitalWestie** [comment]
hey, I just submitted a PR for this, but my fork is based off @rasaHQ's fork, still it might be helpful: https://github.com/RasaHQ/chatroom/pull/2
